### PR TITLE
Various changes and fixes

### PR DESCRIPTION
**Pull request by SergeyRuzin**
1. Made log for failed table binding more readable
2. Added sample feature which does not require deployed demo app
3. Fixed enabled check inverted log
4. Changed package-lock dependencies to fix github alerts

### CODE REVIEW

File: src/Behavioral.Automation/Services/PrintValuesHelper.cs [hunk 1]
```diff
@@ -8,7 +8,8 @@ public static class PrintValuesHelper
     {
         public static string GetPrintableValues(this IEnumerable<ITableRowWrapper> rows)
         {
-            return rows.SelectMany(x => x.CellsText).Aggregate((x, y) => $"{x}, {y}");
+            var values = rows.Aggregate("\r\n|", (current, row) => current + (row.CellsText.Aggregate((x, y) => $" {x} | {y}") + " |\r\n |"));
```
Comment (vladimir-s-quantory) [inline src/Behavioral.Automation/Services/PrintValuesHelper.cs:11]: `Aggregate `method doesn't apply for empty or null collections and will throw exception in this case. I guess we should check `rows` for null/empty and throwing a readable exception or empty string in this case

File: src/Behavioral.Automation/Services/PrintValuesHelper.cs [hunk 2]
```diff
@@ -8,7 +8,8 @@ public static class PrintValuesHelper
...
+            return values.Remove(values.Length - 3);
```
Comment (vladimir-s-quantory) [inline src/Behavioral.Automation/Services/PrintValuesHelper.cs:12]: Why do we need to remove the last three characters from `values`?

**Review by vladimir-s-quantory — CHANGES_REQUESTED** [review]

File: src/Behavioral.Automation/Services/PrintValuesHelper.cs (hunk 2 above)
Comment (SergeyRuzin) [reply src/Behavioral.Automation/Services/PrintValuesHelper.cs:12]: It's the last letters with "\r\n |"

File: src/Behavioral.Automation/Services/PrintValuesHelper.cs (hunk 1 above)
Comment (SergeyRuzin) [reply src/Behavioral.Automation/Services/PrintValuesHelper.cs:11]: Added check which returns "Row collection is empty" string

**Review by vladimir-s-quantory — DISMISSED** [review]

**Review by Brrovko — APPROVED** [review]

File: src/Behavioral.Automation/Bindings/DropdownBinding.cs [hunk 3]
```diff
@@ -116,7 +116,7 @@ public void CheckMultipleSelectedValues([NotNull] IMultiSelectDropdownWrapper wr
         {
             foreach (var row in table.Rows)
             {
-                runnerAction($"the \"{row.Values.First()} \" value {behavior} {enabled} in {wrapper.Caption}:");
+                runnerAction($"the \"{row.Values.First()}\" value {behavior} {enabled} in {wrapper.Caption}");
```
Comment (vladimir-s-quantory) [inline src/Behavioral.Automation/Bindings/DropdownBinding.cs:119]: Are we sure that row.Values are not empty?

Comment (SergeyRuzin) [reply src/Behavioral.Automation/Bindings/DropdownBinding.cs:121]: Not really. Added assert to check it

**Review by Brrovko — APPROVED** [review]

**Review by vladimir-s-quantory — APPROVED** [review]

**Pull request merged**